tests/instrumentation/requests_tests.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from requests.exceptions import InvalidURL, MissingSchema
+from urllib3.exceptions import ConnectionError
+
+import opbeat
+import opbeat.instrumentation.control
+from opbeat.traces import trace
+from tests.helpers import get_tempstoreclient
+from tests.utils.compat import TestCase
+
+
+class InstrumentRequestsTest(TestCase):
+    def setUp(self):
+        self.client = get_tempstoreclient()
+        opbeat.instrumentation.control.instrument()
+
+    def test_requests_instrumentation(self):
+        self.client.begin_transaction("transaction.test")
+        with trace("test_pipeline", "test"):
+            try:
+                requests.get('http://example.com')
```

Comment:
is there any way we can mock this? If at all possible, the tests shouldn't hit outside services.